doc/code/qml_qchem.rst

```diff
@@ -172,10 +172,11 @@ OpenFermion-PySCF backend
 -------------------------
 
 The :func:`~.molecular_hamiltonian` function can be also used to construct the molecular Hamiltonian
-with a non-differentiable backend that uses the
-`OpenFermion-PySCF <https://github.com/quantumlib/OpenFermion-PySCF>`_ plugin interfaced with the
+with non-differentiable backends that use the
+`OpenFermion-PySCF <https://github.com/quantumlib/OpenFermion-PySCF>`_ plugin and the
 electronic structure package `PySCF <https://github.com/sunqm/pyscf>`_. The non-differentiable
-backend can be selected by setting ``method='pyscf'`` in :func:`~.molecular_hamiltonian`:
+backends can be selected by setting ``method='openfermion'`` and ``method='pyscf'``
+in :func:`~.molecular_hamiltonian`:
```

Comment:
Optional: since we already linked the function just a couple of lines above, it breaks the visual flow of the text - 
```suggestion
in ``molecular_hamiltonian``:
```